packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -95,127 +75,41 @@ class VisualRegressionLauncher {
    */
-  async onComplete(_exitCode, _config, _capabilities) {
-    await this.runHook('onComplete');
-  }
-
-  // eslint-disable-next-line consistent-return
-  async runHook(hookName, ...args) {
-    if (typeof this.compare[hookName] === 'function') {
-      // eslint-disable-next-line no-return-await
-      return await this.compare[hookName](...args);
-    }
-  }
-
-  validateConfig(_config) {
-    if (!_.isPlainObject(this.options) || !_.has(this.options, 'compare')) {
-      throw new Error('Please provide a visualRegression configuration with a compare method in your wdio-conf.js!');
-    }
-  }
-
-  wrapCommand(browser, type, command) {
-    const baseContext = {
-      type,
-      browser: this.context.browser,
-      desiredCapabilities: this.context.desiredCapabilities,
-    };
-
-    const runHook = this.runHook.bind(this);
-
-    const getTestDetails = () => this.getTestDetails();
-
-    const resolutionKeyPlural = browser.isMobile ? 'orientations' : 'viewports';
-    const resolutionMap = browser.isMobile ? mapOrientations : mapViewports;
-
-    const viewportChangePauseDefault = this.viewportChangePause;
-    const resolutionDefault = browser.isMobile ? this.orientations : this.viewports;
-
-    return async function async(...args) {
-      const url = await browser.getUrl();
-
-      const elementSelector = type === 'element' ? args[0] : undefined;
-      const options = _.isPlainObject(args[args.length - 1]) ? args[args.length - 1] : {};
-
-      const { exclude, hide, remove } = options;
-
-      const resolutions = _.get(options, resolutionKeyPlural, resolutionDefault);
-      const viewportChangePause = _.get(options, 'viewportChangePause', viewportChangePauseDefault);
-
-      const results = await resolutionMap(
-        browser,
-        viewportChangePause,
-        resolutions,
-        // eslint-disable-next-line prefer-arrow-callback
-        async function takeScreenshot(currentFormFactor) {
-          const meta = _.pickBy(
-            {
-              url,
-              element: elementSelector,
-              exclude,
-              hide,
-              remove,
-              currentFormFactor,
-            },
-            _.identity,
-          );
-
-          const screenshotContext = {
-            ...baseContext,
-            ...getTestDetails(),
-            meta,
-            options,
-          };
-
-          const screenshotContextCleaned = _.pickBy(screenshotContext, _.identity);
-
-          await runHook('beforeScreenshot', screenshotContextCleaned);
-
-          const base64Screenshot = await command(browser, ...args);
+  wrapCommand(browser, command) {
+    return async function wrappedScreenshotCommand(...args) {
+      let currentFormFactor;
+      if (browser.isMobile) {
+        currentFormFactor = await browser.getOrientation();
+      } else {
+        currentFormFactor = await getTerraFormFactor();
+      }
+
+      const screenshotContext = {
+        browserInfo: this.context.browserInfo,
```

Comment:
I changed this to browserInfo from browser because I was getting confused with the global browser object and this key that references this: https://github.com/cerner/terra-toolkit/pull/467/files#diff-a0b45d0f7e8955cf14ad794ee264f64f4c52cb15ecd2ffde8519d5b906ca5fe1R39